themes/digital.gov/layouts/resources/single.html

```diff
@@ -18,27 +18,22 @@ <h1>{{- .Title | markdownify -}}</h1>
       </header>
 
       <div class="usa-in-page-nav-container">
-        <div>
-          {{/* Content */}}
-          <div class="content usa-prose">
-            {{- .Content -}}
-          </div>
-
-          {{/* Touchpoints Form */}}
-          {{- partial "core/touchpoints-form.html" . -}}
+        <div class="content usa-prose">
+          {{- .Content -}}
         </div>
+
         <aside
           class="dg-sidebar usa-in-page-nav"
           data-title-heading-level="h4"
           data-root-margin="-350px 0px -350px 0px"
           aria-label="Related content"
         >
           <div class="dg-related-items" data-related-items>
-            {{/* Related Communities and Services */}}
             {{- partial "core/get_related.html" . -}}
           </div>
         </aside>
       </div>
+      {{- partial "core/touchpoints-form.html" . -}}
```

Comment:
### Fix

Moved this partial call outside of the `in-page nav` markup, it was breaking the layout.

<img width="1187" alt="Screenshot 2024-08-08 at 11 55 28 AM" src="https://github.com/user-attachments/assets/1939341a-b01e-41f1-b999-179db68f2097">